### PR TITLE
Watermark - Global Meeting Policy

### PR DESCRIPTION
**Pull request by Gautamkv**
Watermark need to be enable under "Global" meeting policy.

### CODE REVIEW

File: Teams/watermark-meeting-content-video.md [hunk 1]
```diff
@@ -76,13 +76,13 @@ For information about using watermarks with other meeting features to help prote
 
 ## Enable watermarks
 
-For watermarks to be available in templates and sensitivity labels, and to the meeting organizer, they must be enabled in the Teams admin center.
+For watermarks to be available in templates and sensitivity labels, and to the meeting organizer, they must be enabled under "Global" Meeting policy in the Teams admin center.
```
Comment (JohanFreelancer9) [inline Teams/watermark-meeting-content-video.md:79]:
```suggestion
For watermarks to be available in templates and sensitivity labels, and to the meeting organizer, they must be enabled under the **Global** meeting policy in the Teams admin center.
```

File: Teams/watermark-meeting-content-video.md [hunk 2]
```diff
@@ -76,13 +76,13 @@ For information about using watermarks with other meeting features to help prote
...
 
 To enable watermarking for meetings
 
 1. In the Teams admin center, expand **Meetings** and select **Meeting policies**.
 
-1. Select the policy you want to update.
+1. Select the "Global" policy to update.
```
Comment (JohanFreelancer9) [inline Teams/watermark-meeting-content-video.md:85]:
```suggestion
1. Select the **Global** policy to update.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Gautamkv, kindly commit the minor suggested changes. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**